### ISSUE TITLE
ЧПУ

### ISSUE THREAD

**Issue by Suvitruf** [razrabs-media/journal#282]
Хорошо бы сделать адреса ссылок более читаемыми. В этом плане очень хорошо сделано в WP. К id'шнику/хешу можно добавлять транслит заголовка.

Скажем, для "Разраб — кто ты, что с тобой стало, что с тобой будет" будет razrab-kto-ty-chto-s-toboj-stalo-chto-s-toboj-budet. А если ещё как префикс добавлять дату, то вообще топчик.

**Comment by batyshkaLenin** [comment]
Нужно учесть, что при переходе на ЧПУ нужно будет ещё сделать миграцию для поисковых систем (уведомить их, что адреса статей изменились)

**Comment by Suvitruf** [comment]
> Нужно учесть, что при переходе на ЧПУ нужно будет ещё сделать миграцию для поисковых систем (уведомить их, что адреса статей изменились)

Да. Поэтому, если и планируется переход на ЧПУ, то лучше делать сейчас, пока статей мало.
